src/pnld-download/index.ts

```diff
@@ -4,6 +4,20 @@ import consistentSort from "../lib/consistentSort"
 import config from "./config"
 import PnldFileDownloader from "./PnldFileDownloader"
 import processPnldFiles from "./processPnldFiles"
+import mapOffenceCodeData from "../map-data/mapOffenceCodeData"
+
+type OffenceCodeRecords = {
+  cjsCode: string
+  description?: string
+  homeOfficeClassification?: string | null
+  notifiableToHo?: string
+  recordCreated?: number[]
+  source?: string
+  offenceCategory?: string
+  offenceTitle?: string
+  recordableOnPnc?: string
```

Comment:
boolean?

Reply:
Fixed in [f7e39d6](https://github.com/ministryofjustice/bichard7-next-data/pull/45/commits/f7e39d6e8dfd2e8c8467f4905fe26058a1f0dfe2)